pennylane/templates/layers.py

```diff
@@ -12,22 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 r"""
-This module contains templates for trainable 'layers' of quantum gates.
-
-In contrast to other templates such as embeddings, layers
-do typically only take trainable parameters, and get repeated in the circuit -- just like the layers of a
-neural network. This makes the layer 'learnable' within the limits of the architecture.
-
-Most templates in this module have a 'Layer' version that implements a single layer, as well as a 'Layers'
-version which calls the single layer multiple times, possibly using different hyperparameters for the
-sequence in each call.
+Layers are trainable templates that are typically repeated,
+using different adjustable parameters in each repetition.
```

Comment:
To distinguish between layers and embeddings, perhaps we mention that while embeddings are used to transfer classical data into a quantum state, layers are typically used to carry out some transformation or computation *on a* quantum state? (i.e., Embedding: classical->quantum, Layer: quantum->quantum)